### PR TITLE
Fixed several typos

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
I was reading this page of docs and found a few typos, so take a look @bogensberger or @seut and hopefully you'll agree with them :)

### CODE REVIEW

File: docs/storage_consistency.txt [hunk 1]
```diff
@@ -53,11 +53,11 @@ are synchronous over all active replicas with the following flow:
 4. If the operation succeeds on the primary, the operation gets
    executed on all replicas in parallel.
 
-5. After all replica operations are finished the operation result gets
+5. After all replica operations finish the operation result gets
    returned to the caller.
 
 Should any replica shard fail to write the data or times out in step
-5, it is immediatly considered as unavailable.
+5, i's immediatly considered as unavailable.
```
Comment (mfussenegger) [inline docs/storage_consistency.txt]: `i's` ?

**Comment by ChrisChinchilla** [comment]
fixed @mfussenegger

**Comment by mfussenegger** [comment]
ack,

can you squash and merge it?

**Comment by ChrisChinchilla** [comment]
Done @mfussenegger

**Pull request merged**